mk/build.mk

```diff
@@ -83,6 +83,10 @@ $(foreach target,$(BUILD_RELEASE_BINARIES) $(BUILD_TEST_BINARIES),$(eval $(call
 # Build_Go_Application is a build command for the Kuma Go applications.
 Build_Go_Application = GOOS=$(1) GOARCH=$(2) $$(GOENV) go build -v $$(GOFLAGS) $$(LD_FLAGS) -o $$@/$$(notdir $$@)
 
+build/iptables-wrapper:
+	git clone https://github.com/kubernetes-sigs/iptables-wrappers.git --single-branch $(@)
+	git -C $(@) checkout 5792812d9e5a5bb7f22d79d557bbfeece253343d # latest commit on master (March 25, 2024)
```

Comment:
We add a lot of issues with random git clones in our builds. Is there anyway we can work around this?

Reply:
We can add it as a git submodule

Reply:
Can we fork it and add the binaries to a release?

Reply:
@bartsmykla can we do the fork thing?

Reply:
Could you elaborate on what do you mean by putting in the release? It's binary nad shell script which needs to be placed in the container

Reply:
Done